### PR TITLE
fix: Accessibility SBBTextArea and SBBTextField

### PR DESCRIPTION
**Pull request by fleuryj**

### CODE REVIEW

File: ci_scripts/__Snapshots__/Views/SBBTextFieldTests/testTextFieldPlaceholderNoText.1.png
Comment (fvessaz) [inline ci_scripts/__Snapshots__/Views/SBBTextFieldTests/testTextFieldPlaceholderNoText.1.png:1]: Looks weird... is it normal so?

File: ci_scripts/__Snapshots__/Views/SBBTextFieldTests/testTextFieldPlaceholderNoText.2.png
Comment (fvessaz) [inline ci_scripts/__Snapshots__/Views/SBBTextFieldTests/testTextFieldPlaceholderNoText.2.png:1]: Looks also weird...

**Review by fvessaz — COMMENTED** [review]
The 2 screenshots seems strange.... is it normal so?

File: ci_scripts/__Snapshots__/Views/SBBTextFieldTests/testTextFieldPlaceholderNoText.1.png
Comment (fleuryj) [reply ci_scripts/__Snapshots__/Views/SBBTextFieldTests/testTextFieldPlaceholderNoText.1.png:1]: Nope, not normal, i'm looking into it

**Pull request merged**